public/js/components/Editor.js

```diff
@@ -221,22 +223,30 @@ const Editor = React.createClass({
     }
   },
 
-  render() {
+  renderBreakpoints() {
     const { breakpoints, sourceText } = this.props;
     const isLoading = sourceText && sourceText.get("loading");
 
+    if (isLoading) {
+      return;
+    }
+
+    return breakpoints.valueSeq().map(bp => {
+      return Breakpoint({
+        key: makeLocationId(bp.location),
+        breakpoint: bp,
+        editor: this.editor && this.editor.codeMirror
+      });
+    });
+  },
+
+  render() {
     return (
       dom.div(
         { className: "editor-wrapper devtools-monospace" },
         dom.div({ className: "editor-mount" }),
-        !isLoading &&
-          breakpoints.valueSeq().map(bp => {
-            return Breakpoint({
-              key: makeLocationId(bp.location),
-              breakpoint: bp,
-              editor: this.editor && this.editor.codeMirror
-            });
-          })
+        this.renderBreakpoints(),
```

Comment:
It's just different styles. I generally like to keep as much markup in one place as possible because I find it less syntax visually and easier to read through. There's different styles in the React community, many are like that but other like to break things up too. It's not a huge deal.